AirCasting/SessionViews/StreamsView.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 29/06/2021.
+//
+
+import SwiftUI
+
+struct StreamsView: View {
+    
+    @Binding var selectedStream: MeasurementStreamEntity?
+    @ObservedObject var session: SessionEntity
+    var threshold: SensorThreshold
```

Comment:
Can this be private too?

Reply:
Unfortunately nope, because StreamsView is then inaccessible in AirMapViewm GraphView and SessionCartView :)